test/conformance/enqueue/urEnqueueMemImageWrite.cpp

```diff
@@ -2,3 +2,143 @@
 // Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
 // See LICENSE.TXT
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#include <uur/fixtures.h>
+
+using urEnqueueMemImageWriteTest = uur::urMemImageQueueTest;
+UUR_INSTANTIATE_DEVICE_TEST_SUITE_P(urEnqueueMemImageWriteTest);
+
+TEST_P(urEnqueueMemImageWriteTest, Success1D) {
+    std::vector<uint32_t> output(width * 4, 42);
```

Comment:
Is this `*4` because of the four channels?

Reply:
It is, I'll add a comment to make that clearer